pyfat/PyFAT.py

```diff
@@ -0,0 +1,143 @@
+
+    #################################################
+    #   Python FATigue Test Data Analysis Package   #
+    #                  Version 1.0                  #
+    #                                               #
+    #                  Kevan Gahan                  #
+    #################################################
+
+#-----------------------#
+from pathlib import Path
+import os
+import argparse
+#--------#
+import monotonic
+import get_channels
+#-----------------------#
+
+
+def io_sorter(input_file):
+    """Reads the input file for the data input location and the 
+    output location (where to save results to)."""
+
+    with open(Path(input_file),'r') as input_file:
+        lines = input_file.readlines()
+
+    for line in lines:
+        #Remove and spaces...
+        std_line = line.replace(" ","")
+        #Assign file paths...
+        if std_line.startswith("INPUT="):
+            input_loc = std_line[6:]
+        elif std_line.startswith("OUTPUT="):
+            output_loc = std_line[7:]
+        elif std_line.startswith("#") or std_line.startswith(""):
```

Comment:
If you are only looking for input and output I am not sure you really need this elif statement